pyproject.toml

```
@@ -1,15 +1,58 @@
 [build-system]
+build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools >= 40.0",
+  "setuptools",
+  "setuptools-scm",
 ]
-build-backend = "setuptools.build_meta"
 
+[tool.check-manifest]
+ignore = [".pre-commit-config.yaml"]
 
+[project]
+name = "pytest-rerunfailures"
+version = "13.1.dev0"
+description = "pytest plugin to re-run tests to eliminate flaky failures"
+readme = "README.rst"
+keywords = [
+  "failures",
+  "flaky",
+  "pytest",
+  "pytest",
+  "rerun",
+]
+license = {text = "Mozilla Public License 2.0 (MPL 2.0)"}
+authors = [{name = "Leah Klearman", email = "lklrmn@gmail.com"}]
+requires-python = ">=3.8"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Framework :: Pytest",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Topic :: Software Development :: Quality Assurance",
+  "Topic :: Software Development :: Testing",
+  "Topic :: Utilities",
+]
+dependencies = [
+  "packaging>=17.1",
+  "pytest>=7",
+]
+urls = {Homepage = "https://github.com/pytest-dev/pytest-rerunfailures"}
+[project.entry-points.pytest11]
+pytest-rerunfailures = "pytest_rerunfailures"
```

Comparing before and after sdists, looks like we had no prefix before:

```suggestion
rerunfailures = "pytest_rerunfailures"
```